app/client/templates/components/app-rating.scss

```diff
@@ -36,6 +36,9 @@
       color: #783189;
       font-size: 1.35rem;
       font-weight: 400;
+      @media (max-width: 350px) {
```

Comment:
It strikes me as a little sad that we're not using the existing breakpoints here, but I can deal with it!

It might merit a comment explaining why.

Not a show-stopper.